backend/Private_Messaging/admin.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib import admin
+from .models import Conversation
+
+# Register your models here.
+admin.site.register(Conversation)
```

Comment:
Should messages also be included here? In the future, we could remove it to keep it more private, but if you don't add it, we will need to implement a component in the frontend to both display the messages as well as create the messages. I think it would be easier to just give access to messages in the admin view and remove it later.

Reply:
Yeah, I can add messages just in case